docs/toml.md

```diff
@@ -799,7 +799,7 @@ swarmmode = false
 
 Labels can be used on containers to override default behaviour:
 
-- `traefik.backend=foo`: assign the container to `foo` backend
+- `traefik.backend=foo`: give the name `backend-foo` to the generated backend for this container.
```

Comment:
I'd say that's correct, even though the `backend-` prefix is just something the Docker template does and could be changed by users employing a customized template.